storage/boltdb/client_test.go

```diff
@@ -78,3 +80,53 @@ func TestSuiteShared(t *testing.T) {
 		testsuite.RunTests(t, store)
 	}
 }
+
+type boltLongBenchmarkStore struct {
+	*Client
+	dirPath string
+}
+
+func (store *boltLongBenchmarkStore) BulkImport(iter storage.Iterator) (err error) {
+	// turn off syncing during import
+	oldval := store.db.NoSync
+	store.db.NoSync = true
+	defer func() { store.db.NoSync = oldval }()
+
+	var item storage.ListItem
+	for iter.Next(&item) {
+		if err := store.Put(item.Key, item.Value); err != nil {
+			return fmt.Errorf("Failed to insert data (%q, %q): %v", item.Key, item.Value, err)
+		}
+	}
+
+	return store.db.Sync()
+}
+
+func (store *boltLongBenchmarkStore) BulkDelete() error {
+	// just delete all the things!
+	return os.RemoveAll(store.dirPath)
```

Comment:
Doh, good point. It's been many years since I wrote anything targeting windows. I suppose this should be a no-op function, then, because everything will get cleaned up afterward in this boltdb case.